template/function/serverless.yml

```diff
@@ -1,21 +1,29 @@
-service: app
+service: TPL_SERVICE_NAME
 
 provider:
-    name: aws
-    region: us-east-1
-    runtime: provided.al2
+  # general settings
+  name: aws
+  stage: dev
+  region: TPL_REGION
+  profile: default
+  deploymentMethod: direct
+  # global function settings (apply to each declared function)
+  runtime: provided.al2
+  memorySize: 1024
+  timeout: 6
```

Comment:
Both of these options are already the default value, I think if we remove these it will be less overwhelming for new users.